hera_qm/xrfi.py

```diff
@@ -735,7 +735,14 @@ def xrfi_apply(filename, args, history):
     else:
         raise ValueError('Unrecognized output file format ' + str(args.outfile_format))
     if args.output_npz:
-        # Save an npz with the final flag array and waterfall
+        # Save an npz with the final flag array and waterfall and relevant metadata
         outpath = outpath + args.out_npz_ext
-        np.savez(outpath, flag_array=uvd.flag_array, waterfall=wf_full,
-                 history=flag_history + history)
+        lsts = []
+        for l in uvd.lst_array.ravel():
```

Comment:
does np.unique not work for uvd.lst_array?

Reply:
nope... that reorders it when it could be wrapped inherently. think about an lstarray that is [4, 4.5, 5, 5.5, 6, 0.25, ...]